workspaces/optic-domain/src/shapes.ts

```diff
@@ -30,6 +30,8 @@ export enum JsonLike {
   UNDEFINED = 'Undefined',
 }
 
+export type JsonType = JsonLike;
```

Comment:
Why do we need this? Can't we use the enum directly as a type?

Reply:
I think there's some type juggling that you can do to convert an enum into a type union (I spend a bit of time digging into this before going with a different approach) if this is what you were trying to do:
https://stackoverflow.com/a/52396706

Reply:
Yay, Typescript 🥲.

Reply:
Hmm if it's just for renaming, we can just find replace in all typescript files `JsonLike` -> `JsonType` - or change optic domain and follow the trail where typescript starts yelling

Reply:
Renamed in https://github.com/opticdev/optic/pull/1077/commits/84241850d5f563147868d290f020588563ea6f41